### DISCUSSION TITLE
[Support]: Would uploading a certain number to plus(and het the model)  of false positives allow to remove a object mask

### DISCUSSION THREAD

**Discussion by ronbruins** [blakeblackshear/frigate#10468] [General Support]
### Describe the problem you are having

I have a camera on our front door pointed at a place where we put out bikes. Frigate keeps detecting a bike as a person, whenever there is (other) motion, I already have an object mask that should prevent this, but as the bike moves when it is used, I have to modify the mask and the area also is meant to actually detect persons, so I don't want to have a too large mask. 

Would uploading a certain amount of false positives to my frigate+ and then created a new model, prevent this bike from being detected as a person?

### Version

latest

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Frigate stats

```json
NA
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

reolink

**Comment by nwish** [comment]
I’d say there’s a good chance.  A lot of people are reporting success in that regard with Frigate+, myself included.  You’d essentially be teaching your model the the bikes are not people.

**Reply by ronbruins** [reply]
So in theory, anything that gives a constant amount of false positives, when used to upload to F+ it would allow to get rid of object masks. Any idea how many images would train the model accordingly?

**Reply by blakeblackshear** [reply]
It's going to vary significantly from user to user. True positives near the bike will also help. I would suggest reading this if you haven't already: https://docs.frigate.video/plus/improving_model

**Reply by ronbruins** [reply]
Yeah read that and that's what I have been doing mainly for our cats who now after using a trained model are not identifying as dogs anymore (I hope they are ok with that ;-) ) I didn't had true positives as the bike is pretty much in the front, so difficult to get true positives. I just kicked off another model request, so let's see what this one does and I'll take it from there. Thanks for the answers.

**Reply by nwish** [reply]
You could also do these things:
- trigger true positives near the bikes by walking around there and submit then to Frigate+ along with your false positives
- use min/max area and/or min/max ratios if applicable - for this to work, the bounding boxes for the bike(s) will need to be different in pixel size and ratio than the person boxes
